apps/src/lab2/projects/ProjectManager.ts

```diff
@@ -91,7 +91,11 @@ export default class ProjectManager {
     }
 
     this.lastChannel = channel;
-    return {sources, channel};
+    const response = await HttpClient.fetchJson<{abuse_score: number}>(
```

Comment:
We should move this into the [channelsApi](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lab2/projects/channelsApi.ts) and reference it from the [channelsStore](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lab2/projects/ChannelsStore.ts) for consistency

Reply:
Makes sense - updated!